config/webpack.config.js

```diff
@@ -36,6 +36,11 @@ module.exports = function(env) {
 	// Load applicable .env files into environment variables.
 	require('./dotenv').load(app.context);
 
+	// Determine dynamic iLib location for aliasing
+	const iLib = ['ilib', '@enact/i18n/node_modules/ilib'].find(f =>
```

Comment:
Should we use the same list of directories as in the jest config? Also, I'm curious when `@enact/i18n/node_modules/ilib` would be valid.

Reply:
The jest config allows for both compatibility aliases simulataeously, whereas webpsack does not. Plus Jest includes code to set the `ILIB_BASE_PATH` env var and handle in-library tests, so more verbose detection is needed there, whereas in webpack we can reduce to a simple '@enact/i18n/ilib` path and let webpack do the resolving.

As for `@enact/i18n/node_modules/ilib`, I think you're right and under the peer dependency model, we should be able to drop it.  Any symlink scenario would still reflect peer dependency and require the top-level app to have an `ilib` dependency.

Reply:
Just occurred to me that we _might_ need it for unit testing.

Reply:
Hmm, for jest, moonstone tests and i18n tests both have devDependency of `ilib`, so I believe they should still resolve to `node_modules/ilib`

Reply:
Perfect. That's best.